psen_scan_tutorials/launch/two_scanners.launch

```diff
@@ -0,0 +1,27 @@
+<!--
+Copyright (c) 2019 Pilz GmbH & Co. KG
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-->
+<!-- %Tag(FULLTEXT)% -->
```

Comment:
What's this comment "FULLTEXT" about?

Reply:
Those are tags.
I wanted to use a similar technique as in this [Tutorial](http://wiki.ros.org/roscpp_tutorials/Tutorials/WritingPublisherSubscriber#The_Code).
The tagged code for the above tutorial can be found [here](https://raw.githubusercontent.com/ros/ros_tutorials/kinetic-devel/roscpp_tutorials/talker/talker.cpp).
It can be embedded in a Tutorial through the Makro [GetTaggedCode](http://library.isr.ist.utl.pt/docs/roswiki/WikiMacros.html#GetCode.2C_GetTaggedCode).

Reply:
Cool, thanks for the link!